TemplateBicep/create_device.sh

```diff
@@ -1,140 +1,47 @@
-create_gateway_device () {
-    echo "Creating gateway device $EDGE_GATEWAY_NAME located at resourceGroup=$RESOURCE_GROUP, IotHub=$IOTHUB_NAME..."
-    az iot hub device-identity create --device-id "$EDGE_GATEWAY_NAME" --am shared_private_key --edge-enabled true --hub-name "$IOTHUB_NAME" --resource-group "$RESOURCE_GROUP"
-
-    # Gets the gateway device module manifest
-    # - in bicep the template will be passed as environment variable
-    # - when running locally we read directly from the file
-    if [ -z "$MODULE_CONFIG" ]; then
-        MODULE_CONFIG=$(cat edgeGatewayManifest.json)
-    fi
-
-    # Replace place holders in module configuration with values
-    local spiDevValue=''
-    if [ ! -z "${SPI_DEV}" ] && [ "$SPI_DEV" != "0" ]; then
-        spiDevValue=",\"SPI_DEV\": { \"value\": \"$SPI_DEV\" }"
-        echo "Setting SPI_DEV to $SPI_DEV"
+resolve_lora_region() {
+    local inputRegion=$1
```

Comment:
As follow up work, could we actually don't do this mapping here and ask the users to give "EU863" so we could actually provision any region supported by the provisioning tool?

Reply:
What should the options be?
- EU863
- US902
- AS923-1
- AS923-2
- AS923-3
- CN470RP1
- CN470RP2

Reply:
Yes and after #1925 gets merged AU915 as well

Reply:
Will tackle in next PR